### DISCUSSION TITLE
Move between two matches on same line with consult-line

### DISCUSSION THREAD

**Discussion by Esnos33** [minad/consult#1104] [Ideas]
Right now when I use consult-line to search for "parrot" in
```
parrot parrot
parrot
```
Consult-line will only let me to move between first and third parrot instance, without letting me move to second instance of parrot. Is there possibility to add this feature? This would be expecially useful in kmacros, where I have to use isearch.

**Comment by minad** [comment]
No, this is not possible. `consult-line` is a line-based search, like `occur`. I suggest to use `Isearch` instead in such cases.

**Reply by Esnos33** [reply]
Ok, I understand. Thank you for answer and for great package, I appreciate it.
